webapp/tests/mocha/unit/views/utils.js

```diff
@@ -5,7 +5,7 @@ const vm = require('vm');
 
 const MAP_ARG_NAME = 'doc';
 
-module.exports.loadView = (ddocName, viewName) => {
+module.exports.loadView = (ddocName, viewName, saveValues = false, reset = false) => {
```

Comment:
These new arguments seems a bit indirect.  It's also not clear to me why it's necessary.  Surely each call to `loadView` re-initialises the view anyway?  Perhaps loading the view in `beforeEach()` for the `describe()` for that function would be clearer.  If it then requires an explicit resetting, could this be done in `afterEach()`?